src/network/network.h

```diff
@@ -36,7 +37,13 @@ struct NetworkHandle {
     void close();
 };
 
-struct ServerHandle : public NetworkHandle {};
+struct ServerHandle : public NetworkHandle {
+    std::chrono::steady_clock::time_point heartbeat_time_start;
```

Comment:
Maybe using this name, it's kinda a mouthfull..

Reply:
Yeah I agree, `using steady_tp = std::chrono::steady_clock::time_point`?

Reply:
Sounds great!